entity-framework/core/modeling/data-seeding.md

```diff
@@ -29,6 +29,8 @@ As an example, this will configure seed data for a `Blog` in `OnModelCreating`:
 
 [!code-csharp[BlogSeed](../../../samples/core/Modeling/DataSeeding/DataSeedingContext.cs?name=BlogSeed)]
 
+Notice that you can set the primay key value even for idetity fields (values generated by database) 
+
```

Comment:
A more correct statement is already present in the document, in [line 54](https://github.com/aspnet/EntityFramework.Docs/pull/1219/files#diff-b1227aee2e4d1aacaef6ef7bcf70d1d9L54):

> * The primary key value needs to be specified even if it's usually generated by the database. It will be used to detect data changes between migrations.

Isn't this enough?